classes/classes/Items/Mutations.as

```diff
@@ -3890,7 +3890,7 @@
 					else if (blaht <= 8) player.skinTone = "cerulean";
 					else player.skinTone = "emerald";
 					outputText(player.skinTone + "!");
-					restoreArms(null, RESTOREARMS_FROMGOOSKINTF);
+					if (player.armType != ARM_TYPE_HUMAN || player.clawType != CLAW_TYPE_NORMAL) restoreArms(tfSource);
```

Comment:
Don't fix, what isn't broken, meaning: Don't call restoreArms here, if you have human arms with 'normal' claws aka fingernails